src/SIL.XForge.Scripture/ClientApp/src/app/translate/editor/lynx/insights/quill-services/quill-editor-ready.service.spec.ts

```
@@ -0,0 +1,89 @@
+import { TestBed } from '@angular/core/testing';
+import { firstValueFrom } from 'rxjs';
+import { EditorReadyService } from '../base-services/editor-ready.service';
+import { QuillEditorReadyService } from '../quill-services/quill-editor-ready.service';
+
+class MockQuillEditor {
+  private eventHandlers: Record<string, Function[]> = {};
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)